manifest.json

```diff
@@ -23,16 +23,16 @@
     "type": "object",
     "properties": {
       "decimals": {
-        "title": "admin/application-settings.decimals.title",
+        "title": "Number of decimals",
         "type": "number",
         "default": 2,
-        "description": "admin/application-settings.decimals.description"
+        "description": "Set the number of decimals you want the prices to show"
       },
       "pricesWithTax": {
-        "title": "admin/application-settings.prices-with-tax.title",
+        "title": "Prices with tax",
         "type": "boolean",
         "default": false,
-        "description": "admin/application-settings.prices-with-tax.description"
+        "description": "Add tax to the price shown on Google"
```

Comment:
Are these keys not working?

Reply:
Correct, intl keys cannot be used in the manifest currently (they are rendered as plain text)